ui/src/personal/Personal.js

```diff
@@ -28,6 +28,7 @@ const useStyles = makeStyles({
   root: { marginTop: '1em' },
 })
 
+const autoKey = 'AUTO'
```

Comment:
Maybe use the same pattern here as the `helpKey`? (`_auto`) in this case. Also, if it will be used outside of this file, it should be a constant in the `consts.js` file, to avoid the "magic numbers"

Reply:
Done. Moved AUTO to `consts.js`.